crates/bevy_mikktspace/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "bevy_mikktspace"
+version = "0.8.0-dev"
+edition = "2021"
+authors = ["Benjamin Wasty <benny.wasty@gmail.com>", "David Harvey-Macaulay <alteous@outlook.com>", "Layl Bongers <LaylConway@users.noreply.github.com>"]
+description = "Mikkelsen tangent space algorithm"
+documentation = "https://docs.rs/mikktspace"
+homepage = "https://bevyengine.org"
+repository = "https://github.com/bevyengine/bevy"
+license = "MIT OR Apache-2.0"
```

```suggestion
license = "Zlib AND (MIT OR Apache-2.0)"
```

Since Mikktspace also contains Zlib licensed components, we should make that clear in this field.

---

Done.